lib/features/auth/data/models/user_model.dart

```diff
@@ -14,6 +14,7 @@ class UserModel with _$UserModel {
     required String email,
     required bool isMale,
     required ThemeMode preferredTheme,
+    required WeightUnit preferredWeightUnit,
```

Comment:
This enum is never declared in the PR

Reply:
changed this enum to the Units enum that was already present